docs/transaction-logging.md

```diff
@@ -0,0 +1,128 @@
+# Running a node with finalized transaction logging
+
+The node can be started in a configuration where it logs outcomes of finalized blocks in a Postgres database, and create an index by affected account and smart contract.
+This can be used for monitoring incoming transactions on an account.
+Logging into this database is relatively expensive so it is not enabled in the default configuration.
+
+To enable it the following configuration options must be given. We list environment variables here, but equivalent command-line flags are available, see `concordium-node --help` for their names.
+
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING=true`
+  This enables transaction outcome logging. If this is set the remaining options must be set.
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_NAME`, the name of the database to log into. The database must exist, otherwise the node will fail to start.
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_HOST`, host name or IP of the database, e.g., `127.0.0.1`
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_PORT`, port on which the Postgres database is available, e.g., `5432`
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_USERNAME`, username to log into the database
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_PASSWORD`, password to log into the database
+
+As mentioned above the database must exist, otherwise the node will fail to start. If correct tables exist in the database then they will be used, otherwise the following will be executed upoon startup
```

Comment:
```suggestion
As mentioned above the database must exist, otherwise the node will fail to start. If correct tables exist in the database then they will be used, otherwise the following will be executed upon startup
```